client/finality-grandpa/rpc/src/lib.rs

```diff
@@ -469,7 +469,9 @@ mod tests {
 
 		// Notify with a header and justification
 		let justification = create_justification();
-		justification_sender.notify(|| Ok(justification.clone())).unwrap();
+		let r: Result<(), finality_grandpa::Error> =
+			justification_sender.notify(|| Ok(justification.clone()));
+		r.unwrap();
```

Comment:
```suggestion
		justification_sender.notify(|| Ok::<_, ()>(justification.clone()))
			.expect("only returns Err if the closure fails; the closure always returns Ok; qed.");
```

This code should have had a proof before, we don't use unwrap.

Reply:
This is tests, we `unwrap` in unit-tests.

Reply:
Oops sorry, didn't notice it.